torch/csrc/jit/runtime/register_special_ops.cpp

```diff
@@ -433,6 +456,58 @@ RegisterOperators reg({
         "aten::set_grad_enabled(bool val) -> ()",
         [](Stack* stack) { torch::GradMode::set_enabled(pop(stack).toBool()); },
         aliasAnalysisConservative()),
+    Operator(
+        "aten::set_grad_enabled(bool val) -> ()",
+        [](Stack* stack) { torch::GradMode::set_enabled(pop(stack).toBool()); },
+        aliasAnalysisConservative()),
+    Operator(
+        "aten::getCudaStream(int64_t val) -> __torch__.torch.classes.cuda.Stream",
```

Comment:
I think we should put these in a different namespace if possible (e.g. `cuda`) since they're not really `aten` ops. I think it should be simple since we have full control over how the ops are emitted but there *might* be issues with operator resolution.